metadata.go

```diff
@@ -192,6 +200,86 @@ func GetImageToMDMap(imageMDs []ImageMetadataInfo) (imageToMDMap map[string][]Im
 	return
 }
 
+// getLocalImages queries the local Docker daemon for list of images.
+func getLocalImages() (imageMap map[ImageIDType][]RepoTagType, e error) {
+
+	// query a list of images from Docker daemon
+	response, e := listImages()
+	if e != nil {
+		return nil, e
+	}
+	blog.Info(string(response))
+	// parse JSON
+	var localImageList []LocalImageStruct
+	if e = json.Unmarshal(response, &localImageList); e != nil {
+		return nil, e
+	}
+
+	// make map from each imageID to all of its aliases (repo+tag)
+	imageMap = make(map[ImageIDType][]RepoTagType)
+	for _, localImage := range localImageList {
+		imageID := ImageIDType(localImage.ID)
+		for _, repoTag := range localImage.RepoTags {
+			// skip images with no repo:tag
+			if repoTag == "" {
```

Comment:
Shall we also skip images either with none repo or none tag, in addition to none:none ?